jaas/command/src/main/java/org/apache/karaf/jaas/command/ModuleAddCommand.java

```diff
@@ -0,0 +1,110 @@
+package org.apache.karaf.jaas.command;
+
+import org.apache.karaf.jaas.config.JaasRealm;
+import org.apache.karaf.jaas.config.impl.Config;
+import org.apache.karaf.jaas.config.impl.Module;
+import org.apache.karaf.jaas.modules.BackingEngine;
+import org.apache.karaf.shell.api.action.Argument;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Properties;
+
+@Command(scope = "jaas", name = "module-add", description = "Add a Login Module")
+@Service
+public class ModuleAddCommand extends JaasCommandSupport {
+
+    @Argument(index = 0, name = "loginModule", description = "Class Name of Login Module", required = true, multiValued = false)
+    private String loginModule;
+
+    @Argument(index = 1, name = "properties", description = "Pair of Properties (key value)", required = false, multiValued = true)
+    private List<String> propertiesList;
+
+    @Override
+    protected Object doExecute(BackingEngine engine) throws Exception {
```

Comment:
Why extending `JaasCommandSupport` if we don't use the method ? I would directly implement `Action`.